spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -75,6 +74,18 @@ public void publishMessage() {
 }
 ----
 
+
+The default serialization and deserialization of POJOs uses Jackson JSON via the `JacksonMessageConverter` class.
+However, the Jackson dependency is required to utilize that implementation:
```

Comment:
That's something wrong.
People are going to argue that we require Jackson to use with the `PubSubTemplate`, but a dependency on the matter is not included.
Or let's remove `optional` from there, or don't instantiate Jackson converter in the `PubSubTemplate`. The dependency between our classes eliminates an optional nature of the external dependency.

Reply:
Ah, that's right, required is fine then.

Reply:
We can come back to `optional` later, when we will be able to introduce some other converter implementation. However we can't regain it to the `optional` until the next major `2.0` version though :-)

Reply:
@meltsufin done, PTAL